Library/Formula/source-highlight.rb

```diff
@@ -2,10 +2,10 @@
 
 class SourceHighlight < Formula
   homepage 'https://www.gnu.org/software/src-highlite/'
-  url 'http://ftpmirror.gnu.org/src-highlite/source-highlight-3.1.7.tar.gz'
-  mirror 'https://ftp.gnu.org/gnu/src-highlite/source-highlight-3.1.7.tar.gz'
-  mirror 'http://mirror.anl.gov/pub/gnu/src-highlite/source-highlight-3.1.7.tar.gz'
-  sha1 '71c637548be71afc3f895b0d8ada1a72a8dab4a0'
+  url 'http://ftpmirror.gnu.org/src-highlite/source-highlight-3.1.8.tar.gz'
+  mirror 'https://ftp.gnu.org/gnu/src-highlite/source-highlight-3.1.8.tar.gz'
+  mirror 'http://mirror.anl.gov/pub/gnu/src-highlite/source-highlight-3.1.8.tar.gz'
+  sha1 '55b814ec09674ef7f5ab42c46b2ae5a0f86f7c9b'
```

Comment:
@DomT4 happy that my commits can potentially enable **any** kind of rolling, especially the happy kind :) …formula updated with the output of `shasum -a 256 source-highlight-3.1.8.tar.gz`